src/components/WSLIntegration.vue

```diff
@@ -1,23 +1,19 @@
 <template>
-  <section>
-    <h3 v-text="description" />
-    <section class="body">
-      <ul data-test="integration-list">
-        <li
-          v-for="item of integrationsList"
-          :key="item.name"
-          :data-test="`item-${item.name}`"
-        >
-          <checkbox
-            :value="item.value"
-            :label="item.name"
-            :disabled="item.disabled"
-            :description="item.description"
-            @input="toggleIntegration(item.name, $event)"
-          />
-        </li>
-      </ul>
-    </section>
+  <section class="wsl-integrations">
+    <h3 v-if="description" v-text="description" />
+    <div
```

Comment:
Consider adding back the left margin in the list; currently it's a bit hard to read.